docs/public-networks/reference/cli/options.md

```diff
@@ -5679,6 +5679,52 @@ content if the save and restore functionality is enabled using
 The file is created on shutdown and reloaded during startup.
 The default file name is `txpool.dump` in the [data directory](#data-path).
 
+### `use-cached-genesis-state-hash`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--use-cached-genesis-state-hash=[=<true|false>]
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+<TabItem value="Example configuration file" label="Example configuration file"> 
+
+```bash
+use-cached-genesis-state-hash=true
+```
+
+</TabItem>
+
+</Tabs>
+
+Enables fast startup from an existing genesis state hash. The default is `false`.
+
+:::danger
```

Comment:
```suggestion
:::warning
```

We usually don't use the [red danger](https://docusaurus.io/docs/3.0.1/markdown-features/admonitions) admonition except for the most critical security warnings.